src/host.rs

```diff
@@ -588,6 +589,27 @@ impl Plugin for PluginInstance {
     fn get_parameter_object(&mut self) -> Arc<dyn PluginParameters> {
         Arc::clone(&self.params) as Arc<dyn PluginParameters>
     }
+
+    fn get_editor_rect(&self) -> Option<editor::Rect> {
+        let mut rect: *mut editor::Rect = std::ptr::null_mut();
+        let rect_ptr: *mut *mut editor::Rect = &mut rect;
+
+        self.dispatch(plugin::OpCode::EditorGetRect, 0, 0, rect_ptr as *mut c_void, 0.0);
+
+        if rect != std::ptr::null_mut() {
+            Some(unsafe{ *rect }) // TODO: Who owns rect? Who should free the memory?
```

Comment:
Since `rect` is a pointer to a heap allocated `editor::Rect`, I presume the plugin is going to allocate the memory and set it to the correct value, but I'm not sure whether or when should we free the memory.

Appreciate any advice.